MAINTAINERS.adoc

```diff
@@ -0,0 +1,129 @@
+* Updates the CHANGELOG.adoc file.  This moves the currently-unreleased changes to a new section for the version that you're publishing and sets us up with a new "Unreleased changes" section.
+
+Here's how to cut a release.
+
+. Check that the CHANGELOG.adoc file is up to date for this release.
+.. Any breaking changes should have text that concisely summarizes for a Dropshot user how they know if they're affected and what they need to do to safely upgrade their code.  A few minutes here can save many people many minutes later!  This is best done by updating CHANGELOG.adoc with each PR, rather than all at once when we want to cut a release.
+.. Any other notable changes should be documented.  There's a link in the changelog to a list of unreleased commits -- you can skim these to see if anything looks notable.
+. Double-check your working directory.
+.. `git status` should show that you're on the `main` branch.
+.. Use `git pull` to sync up with any upstream changes.  `git status` should show that your local "main" is exactly sync'd up with the upstream "main" and that you have no uncommitted files or local changes to committed files.
+.. `git clean -nxd` should be clean or close to it.  It should show nothing that should be checked into git and probably no stale build artifacts.
+.. It's a good idea to check that CI completed successfully on GitHub for the commit that you've got checked out.  This will ensure that we're publishing a release that builds from scratch, passes all tests on all platforms, and passes our style and lint checks.
+... Use `git log -1` to see what commit you're on.
+... Look up that commit on GitHub.  It will have a URL like `https://github.com/oxidecomputer/dropshot/commit/COMMIT_SHA` (with an actual `COMMIT_SHA`).  CI succeeded if there's a green check at the left of the commit message.  You can click the green check to be sure you're looking at the right thing.  It should say "All checks have passed".
+. Figure out the tag used for the most recent release.  The easiest is probably to view https://github.com/oxidecomputer/dropshot/releases[the recent releases].   We'll call this `PREV_RELEASE_TAG` in the next step.
+. Figure out what kind of update this is relative to the last one released.  https://github.com/sunng87/cargo-release/blob/master/docs/reference.md#bump-level[See `cargo release` docs for the available levels.]  We're usually doing a "patch", "minor", or "major" release.  We'll call this "RELEASE_KIND".  (According to the docs, you can also use a specific version number that you want to release here.)
+. Do the release.  If you're daring, just run:
++
+[source,text]
+----
+$ cargo release --prev-tag-name=PREV_RELEASE_TAG -vv RELEASE_KIND
```

Comment:
Might be worth mentioning to:

```bash
$ cargo install cargo-release
```

first?

Reply:
Good idea.